content/set-up/clients/authorization-code-client-procedure.md

```diff
@@ -20,7 +20,7 @@ To add an authorization code client:
 
```

Comment:
I noticed two things about the step above:

1. "Client Type dropdown list" is vague, as the dropdown isn't actually named anything. I suggest calling it the "dropdown menu in the upper-left" or something similar.

    ![image](https://github.com/osisoft/Data-Hub-Docs/assets/77757810/021e909c-c5d7-4479-8f7d-53af3948ee04)

2. Rather than "Authorization Code", I think it should be "Authorization Code Clients".